types/src/front/assistant/actions/conversation/list_files.ts

```diff
@@ -7,6 +7,10 @@ export type ConversationFileType = {
   title: string;
   contentType: SupportedContentFragmentType;
   snippet: string | null;
+  isUsableForJIT: boolean;
```

Comment:
This will leak on the API in the events no?

Reply:
Would keep it as a function on ConversationFile

Reply:
> This will leak on the API in the events no?

I don't think so (will check).